package/patterns-yast.spec

```diff
@@ -51,107 +50,124 @@ Provides:       pattern-visible()
 
 Requires:       libyui-ncurses-pkg
 Requires:       yast2
+Requires:       yast2-firewall
+Requires:       yast2-packager
+Requires:       yast2-theme
+
+Recommends:     yast2-metapackage-handler
+Recommends:     yast2-support
+
+Suggests:       yast2-firstboot
+Suggests:       yast2-snapper
+
+# SLE only
+%if !0%{?is_opensuse}
+Requires:       yast2-registration
+# yast2 clone_system is expected to be installed by default (sle-beta)
+Recommends:     autoyast2-installation
+%endif
+
 %if 0%{?is_opensuse}
-# opensuse only, see bsc#1125040
 Requires:       yast2-alternatives
+Recommends:     yast2-vm
```

Comment:
what is reason to recomment yast2-vm only for opensuse?

Reply:
Forgot to transfer the comment about bsc#1083398